dScripts/02_server/Enemy/AM/AmDarklingDragon.cpp

```diff
@@ -62,7 +62,6 @@ void AmDarklingDragon::OnHitOrHealResult(Entity* self, Entity* attacker, int32_t
 			}
 
 			if (skillComponent != nullptr) {
-				skillComponent->Interrupt();
```

Comment:
Why has this been removed?

Reply:
They cannot be interrupted anyways and are reset in the line below